### PR TITLE
[LI-HOTFIX] Add process time breakdown log for LeaderAndIsrRequest

### PR DESCRIPTION
**Pull request by lmr3796**
TICKET = LIKAFKA-46169
LI_DESCRIPTION =
To track down why in 3.0 control requests needs more time to process,
we need logs to track time for each part to process.
EXIT_CRITERIA = When better mechanism to track is implemented upstream

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by lmr3796** [comment]
Sample log line:

```
2022/08/25 11:37:00.827 [logger] [Broker id=0] Breakdown time in milliseconds for LeaderAndIsr request with correlationId 1 from controller 0: Map(validateTopicPartitionInfo -> 40, localLogManagerOnLeadershipChangeCallback -> 2, convertLeaderAndIsrRequestPartitionStates -> 1, constructLeaderAndIsrResponse -> 1, acquireReplicaStateChangeLock -> 0, makeFollowers -> 0, makeLeaders -> 2151, updateLeaderAndFollowerMetrics -> 4, remoteLogManagerOnLeadershipChangeCallback -> 0, findStrayPartition -> 1, deleteStrayReplicas -> 7, generateHighWatermarkCheckpoints -> 1, markPartitionOffline -> 1, startHighWatermarkCheckPointThread -> 0, maybeAddLogDirFetchers -> 0, shutdownIdleFetcherThreads -> 1, validateTopicIdInfo -> 3)
```

File: core/src/main/scala/kafka/server/ReplicaManager.scala [hunk 1]
```diff
@@ -1577,12 +1589,18 @@ class ReplicaManager(val config: KafkaConfig,
                 responseMap.put(topicPartition, error)
               }
             }
+
+            breakdown("validateTopicIdInfo") += time.milliseconds() - validateTopicIdInfoStart
```
Comment (CCisGG) [inline core/src/main/scala/kafka/server/ReplicaManager.scala]: Wouldn't it be the same if you change this line to `breakdown("validateTopicIdInfo") += intervalMarker.markTimeAndReturnLatestInterval(time.milliseconds())` and remove line 1534?

Comment (lmr3796) [reply core/src/main/scala/kafka/server/ReplicaManager.scala]: Fixed.  Good catch

**Review by CCisGG — APPROVED** [review]
lgtm

**Pull request merged**